tests/on_wlm/test_containers_wlm.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+from shutil import which
+
+from smartsim import Experiment, status
+from smartsim.entity import Ensemble
+from smartsim.settings.containers import Singularity
+
+"""Test SmartRedis container integration on a supercomputer with a WLM."""
+
+# Check if singularity is available as command line tool
+singularity_exists = which('singularity') is not None
+
+@pytest.mark.skipif(not singularity_exists, reason="Test needs singularity to run")
+def test_singularity_wlm_smartredis(fileutils, wlmutils):
+    """Run two processes, each process puts a tensor on
+    the DB, then accesses the other process's tensor.
+    Finally, the tensor is used to run a model.
+
+    Note: This is a containerized port of test_smartredis.py for WLM system
+    """
+
+    launcher = wlmutils.get_test_launcher()
+    if launcher not in ["pbs", "slurm"]:
```

Comment:
Change this to just `pbs` for now OR change launchers to `auto` instead of `slurm` below